llpc/context/llpcCompiler.cpp

```diff
@@ -2239,38 +2316,54 @@ void Compiler::LinkRelocatableShaderElf(
     ElfPackage* pPipelineElf,  // [out] Elf package containing the pipeline elf
     Context* pContext)         // [in]  Acquired context
 {
-    LLPC_ASSERT(pShaderElfs[ShaderStageCompute].empty() && "Cannot link compute shaders yet.");
     LLPC_ASSERT(pShaderElfs[ShaderStageTessControl].empty() && "Cannot link tessellation shaders yet.");
     LLPC_ASSERT(pShaderElfs[ShaderStageTessEval].empty() && "Cannot link tessellation shaders yet.");
     LLPC_ASSERT(pShaderElfs[ShaderStageGeometry].empty() && "Cannot link geometry shaders yet.");
 
-    ElfReader<Elf64> vsReader(m_gfxIp);
-    ElfReader<Elf64> fsReader(m_gfxIp);
-    if (!pShaderElfs[ShaderStageVertex].empty())
+    Result result;
```

Comment:
please remove this result, it seems it isn't used.

Reply:
Done. Modified the following lines so this result is used for all calls in this function.